scripts/statcast_timing.py

```diff
@@ -0,0 +1,30 @@
+import sys
+import time
+from pybaseball import statcast
+
+EXPECTED_TIME_THRESHOLD = 30
+START_DATE = "2018-08-01"
+END_DATE = "2018-08-10"
+
+def main():
+    if len(sys.argv) == 2:
+        time_threshold = float(sys.argv[1])
+    else:
+        time_threshold = EXPECTED_TIME_THRESHOLD
+
+    start_time = time.time()
+    _ = statcast(START_DATE, END_DATE)
+    end_time = time.time()
+    query_time = end_time-start_time
+    if query_time > time_threshold:
+        sys.stderr.write("query took {:.2f} seconds "
```

Comment:
Should we use f-strings instead of a format call?

Reply:
yes good idea - I wasn't aware (until I just looked it up now) that you can use a format-code with f-strings.

Reply:
I used part f-strings, part `format`, let me know if loosk o0k

Reply:
It looks okay. You could also do this:

```python
sys.stdout.write(f"query took {query_time: .1f} seconds which is {"less" if query_time < time_threshold else "greater"} than threshold of {time_threshold: .1f}\n")
```

Reply:
For that '{"less" if query_time < time_threshold else "greater"}'
Should be move outside. No long stuff in there